.github/workflows/test-lang-java.yml

```
@@ -63,15 +63,19 @@ jobs:
         with:
           maven-version: 3.9.6
 
+      - name: 'Install Java Avro'
+        working-directory: .
```

Is this needed ?

---

We want to install it from top-level:
```suggestion
        working-directory: /runner/_work/avro/avro
```